app/controllers/events_controller.rb

```diff
@@ -334,7 +334,7 @@ def update
         end
 
 
-        if @event.description_previously_changed?
+        if @event.description_previously_changed? && !@event.description.empty?
```

Comment:
```suggestion
        if @event.description_previously_changed? && @event.description.present?
```
Can you check if this works? I prefer avoiding negation